core/include/thread.h

```diff
@@ -102,13 +102,16 @@ struct _thread {
 #endif
 };
 
- /**
+/**
  * @def THREAD_STACKSIZE_DEFAULT
  * @brief A reasonable default stack size that will suffice most smaller tasks
  */
 #ifndef THREAD_STACKSIZE_DEFAULT
 #error THREAD_STACKSIZE_DEFAULT must be defined per CPU
 #endif
+#ifdef DOXYGEN
+#define THREAD_STACKSIZE_DEFAULT
```

Comment:
I'm just concerned that someone who wants to know the values is running into a dead end because they seem to be defined here, and looking into *thread.h* reveals nothing.
Maybe a hint to `cpu/$CPU/include/cpu_conf.h` in the description could be helpful.